tests/models/test_trigger.py

```diff
@@ -186,61 +199,11 @@ def test_assign_unassigned(session, create_task_instance):
         session.query(Trigger).filter(Trigger.id == trigger_on_healthy_triggerer.id).one().triggerer_id
         == healthy_triggerer.id
     )
-
-
-@pytest.mark.parametrize("check_triggerer_heartrate", [10, 60, 300])
-def test_assign_unassigned_missing_heartbeat(session, create_task_instance, check_triggerer_heartrate):
```

Comment:
A new assert was added to the above test that covers the usecase this tests was trying to cover. This test was quite long and more complicated than I think required to test this particular code branch.